CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - You can now specify deployment tags in the `Cloud Deployment` window.
 - RPCs declared in a UINTERFACE can now be executed. Previously, this would lead to a runtime assertion.
 - When using the `-receptionistHost` command line parameter with a non-local host, it's no longer necessary to set `-useExternalIpForBridge true` as this will be inferred automatically.
+- Support for metrics to be submitted by game code. Note this is only available for already approved Improbable metrics.
```

Comment:
Should we publicize this?

Reply:
Yeah I don't see why not. Should specify what the acceptable metrics are, and potentially create vars for them in `USpatialMetrics`